i18n/english.yml

```diff
@@ -759,6 +764,7 @@ components:
     title: Log in
   ManagerHeader:
     noUpdateYet: n/a
+    bundleFilename: Bundle Filename
```

Comment:
Can you not re-use the string from feed info?

Reply:
annoyingly, the way the internationalization works in DT is that the key is already scoped to ManagerHeader inside the ManagerHeader component, so you can't reach to other strings.